packages/runtime/src/schema.ts

```diff
@@ -451,44 +556,66 @@ class SchemaBuilder {
           },
         }
       }
-      case 'queryConnection':
+      case 'queryConnection': {
+        const qcFiltersObj = this.#inputObjects[`${modelAlias}Filters`]
+        const qcSortingObj = this.#inputObjects[`${modelAlias}Sorting`]
+        const args: ObjMap<GraphQLArgumentConfig> = {
+          ...connectionArgsWithAccount,
+        }
+        if (qcFiltersObj && qcSortingObj) {
+          args.filters = { type: qcFiltersObj }
+          args.sorting = { type: qcSortingObj }
+        }
         return {
           type: new GraphQLNonNull(this.#types[`${modelAlias}Connection`]),
-          args: connectionArgsWithAccount,
+          args,
           resolve: async (
             doc,
-            args: ConnectionArgumentsWithAccount,
+            args: ConnectionRelationArguments,
             ctx
           ): Promise<Connection<unknown> | null> => {
             const account =
               args.account === 'documentAccount' ? doc.metadata.controller : args.account
+            const queryFilters = createRelationQueryFilters(
+              relation.property,
+              doc.id.toString(),
+              args.filters
+            )
             return await ctx.queryConnection({
               ...args,
               account,
               model: relation.model,
-              filter: { [relation.property]: doc.id.toString() },
+              queryFilters,
             })
           },
         }
-      case 'queryCount':
+      }
+      case 'queryCount': {
+        const filtersObj = this.#inputObjects[`${modelAlias}Filters`]
+        const args: ObjMap<GraphQLArgumentConfig> = {
+          account: {
+            type: GraphQLID,
+            description: 'Counts only documents created by the provided account',
+          },
+        }
+        if (filtersObj) {
+          args.filters = { type: filtersObj }
```

Comment:
should this have a `description` field like the `account` arg does?

Reply:
I think @PaulLeCam added this to provide more description on this field, whereas other fields are mostly self explanatory.